src/components/expansion-panel/expansion-panel-body/expansion-panel-body.scss

```diff
@@ -13,6 +13,7 @@
     display: flex;
     justify-content: space-around;
     flex-direction: column;
+    width: 100%;
```

Comment:
The adjustment can't be done in component. The widht need be aply inside the slot.

Reply:
Actually, this has to be done on the component. The slot is already using all the width the parent div (this div) have. The problem we are facing is that the parent div inside the component (this div with class 'with-line') is not occupying all the horizontal space the ExpansionPanelBody has.
The limitation is from the component itself.